examples/servers/simple-resource/pyproject.toml

```diff
@@ -40,7 +40,7 @@ select = ["E", "F", "I"]
 ignore = []
 
 [tool.ruff]
-line-length = 88
+line-length = 120
```

Comment:
Those changes were made to match the main `line-length`. I did it in this PR because when I changed one of the files in the examples, it trigger the formatting.